### PR TITLE
[Preview] Highlight expression yellow #2984

### PR DESCRIPTION
**Pull request by ofiron01**
Associated Issue: #2984 

### Summary of Changes

* changed highlight expression color to more contrast 

image

### Test Plan
N/A

### CODE REVIEW

File: src/components/Editor/Preview.css [hunk 1]
```diff
@@ -28,7 +28,7 @@
 }
 
 .selected-token {
-  background-color: var(--theme-search-overlays-semitransparent);
+  background-color: rgb(255, 255, 180);
   color: var(--theme-selection-color);
```
Comment (bomsy) [inline src/components/Editor/Preview.css]: Nice! It would be cool, if this could be put as part of the theme.

Comment (ofiron01) [inline src/components/Editor/Preview.css]: thanks @bomsy can you point me in the right direction?

Comment (bomsy) [inline src/components/Editor/Preview.css]: @ofiron01 You could look in here to see if we have a variable matching the colour we want, and use that.  https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/src/lib/themes/variables.css#L1

If not, we can do a PR in devtools-core to add a new variable.

**Comment by bomsy** [comment]
@ofiron01 Nice work! Looking good!

**Pull request merged**